Dark.json

```diff
@@ -56105,16 +56104,16 @@
       "set": "Virtual Set 10",
       "front": {
         "title": "•Insignificant Rebellion (V)",
-        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/cards/legacy/Virtual10-Dark/large/insignificantrebellion.gif?raw=true",
+        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/cards/Virtual10-Dark/large/insignificantrebellion.gif?raw=true",
```

Comment:
I get Access Denied when trying to view the updated image url.  Of course, the old one doesn't work either...

Reply:
The non-legacy images are sourced from the Holotable repo.  I verified that relative reference was correct.